ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-accounting.j2

```diff
@@ -11,6 +11,9 @@
 | Exec | - | {{ aaa_accounting.exec.default.type }} | {{ aaa_accounting.exec.default.group }} | - |
 {%        endif %}
 {%     endif %}
+{%     if aaa_accounting.system.default.type is arista.avd.defined and aaa_accounting.system.default.group is arista.avd.defined  %}
+| System | Default | {{ aaa_accounting.system.default.type }} | {{ aaa_accounting.system.default.group }} | - |
```

Comment:
On `exec` above we print `-` and not `Default`. We should align to one or the other.

Reply:
Different options are available for each command, under 4.25.2:
```
POD1_SPINE01(config)#aaa accounting exec ?
  console  Console accounting method list
  default  The default accounting list

POD1_SPINE01(config)#aaa accounting system ?
  default  The default accounting list
```
This can wait until PR#1132 is merged.

Reply:
1132 is merged now, so this one needs to be adjusted to the new documentation layout.